contrib/dropwizard/src/main/java/io/opencensus/contrib/dropwizard/DropWizardMetrics.java

```diff
@@ -0,0 +1,354 @@
+    return collectSnapshotAndCount(name, timer.getSnapshot(), timer.getCount());
+  }
+
+  /**
+   * Collect and transform a timer and histogram snapshot as a opencensus metric.
+   *
+   * @param name metric name
+   * @param snapshot Snapshot object to collect
+   * @param count the value or count
+   * @return a list of metrics.
+   */
+  private List<Metric> collectSnapshotAndCount(String name, Snapshot snapshot, long count) {
+    String metricName = DropWizardUtils.generateFullMetricName(name, "count");
+    String metricDescription = DropWizardUtils.generateFullMetricDescription("Snapshot", "count");
+
+    MetricDescriptor countMetricDescriptor =
+        MetricDescriptor.create(
+            metricName,
+            metricDescription,
+            DEFAULT_UNIT,
+            Type.CUMULATIVE_INT64,
+            Collections.<LabelKey>emptyList());
+    TimeSeries countTimeSeries =
+        TimeSeries.create(
+            Collections.<LabelValue>emptyList(),
+            Collections.singletonList(Point.create(Value.longValue(count), clock.now())),
+            cumulativeStartTimestamp);
+
+    // Collect quantile related metric
+    metricName = DropWizardUtils.generateFullMetricName(name, "quantile");
+    metricDescription = DropWizardUtils.generateFullMetricDescription("Snapshot", "quantile");
+
+    MetricDescriptor quantileMetricDescriptor =
+        MetricDescriptor.create(
+            metricName,
+            metricDescription,
+            DEFAULT_UNIT,
+            Type.GAUGE_DOUBLE,
+            Collections.unmodifiableList(Collections.singletonList(QUANTILE_LABEL_KEY)));
+
+    List<LabelValue> rateLabelValues =
+        Arrays.asList(
+            QUANTILE_50_LABEL_VALUE,
+            QUANTILE_75_LABEL_VALUE,
+            QUANTILE_95_LABEL_VALUE,
+            QUANTILE_98_LABEL_VALUE,
+            QUANTILE_99_LABEL_VALUE,
+            QUANTILE_999_LABEL_VALUE);
+    List<Point> ratePoints =
+        Arrays.asList(
+            Point.create(Value.doubleValue(snapshot.getMedian()), clock.now()),
+            Point.create(Value.doubleValue(snapshot.get75thPercentile()), clock.now()),
+            Point.create(Value.doubleValue(snapshot.get95thPercentile()), clock.now()),
+            Point.create(Value.doubleValue(snapshot.get98thPercentile()), clock.now()),
+            Point.create(Value.doubleValue(snapshot.get99thPercentile()), clock.now()),
+            Point.create(Value.doubleValue(snapshot.get999thPercentile()), clock.now()));
+    TimeSeries quantileTimeSeries = TimeSeries.create(rateLabelValues, ratePoints, null);
+    return Arrays.asList(
+        Metric.create(countMetricDescriptor, Collections.singletonList(countTimeSeries)),
+        Metric.create(quantileMetricDescriptor, Collections.singletonList(quantileTimeSeries)));
+  }
+
+  /** Collect and transform a dropwizard metric as a opencensus metric. */
+  @Override
+  @SuppressWarnings("rawtypes")
+  public Collection<Metric> getMetrics() {
```

Comment:
Public APIs need to have a `@since` tag.

Reply:
This is an override so don't think it needs a since tag.